### PR TITLE
Update bower.md to prefer LibMan

### PR DESCRIPTION
**Pull request by Rick-Anderson**

### CODE REVIEW

File: aspnetcore/client-side/bower.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ uid: client-side/bower
 By [Rick Anderson](https://twitter.com/RickAndMSFT), [Noel Rice](https://blog.falafel.com/falafel-software-recognized-sitefinity-website-year/), and [Scott Addie](https://scottaddie.com) 
 
 > [!IMPORTANT]
-> While Bower is maintained, its maintainers recommend using a different solution. Yarn with Webpack is one popular alternative for which [migration instructions](https://bower.io/blog/2017/how-to-migrate-away-from-bower/) are available.
+> While Bower is maintained, its maintainers recommend using a different solution. [Library Manager](https://blogs.msdn.microsoft.com/webdev/2018/04/18/what-happened-to-bower/) (LibMan for short) is Visual Studio’s new client-side static content management system. Yarn with Webpack is one popular alternative for which [migration instructions](https://bower.io/blog/2017/how-to-migrate-away-from-bower/) are available.
```
Comment (guardrex) [inline aspnetcore/client-side/bower.md]: There's a smart quote there in "Visual Studio's" that should be a normal quote.

**Comment by Rick-Anderson** [comment]
@guardrex  did I get rid of the right smarty quote?

**Comment by Rick-Anderson** [comment]
@guardrex  never mind, missed your comment.

**Comment by guardrex** [comment]
You did! You did! That quote really wasn't all that smart ... I was smarter than that quote ... so how smart could it really have been?? 😀

**Pull request merged**